utils/openal-info.c

```diff
@@ -148,6 +154,10 @@ static void printALCInfo(ALCdevice *device)
         printf("ALC extensions:");
         printList(alcGetString(device, ALC_EXTENSIONS), ' ');
         checkALCErrors(device);
+        printALCIntegerValue(device, ALC_MONO_SOURCES, "ALC_MONO_SOURCES");
+        checkALCErrors(device);
+        printALCIntegerValue(device, ALC_STEREO_SOURCES, "ALC_STEREO_SOURCES");
```

Comment:
This should be done after the context is created and made current. It would also make sense to put the `checkALCErrors` call by the `alc*` calls that could cause an error, and avoid printing the values if there was an error since the variable would be uninitialized in that case.

Reply:
Yes, my bad :x, it's fixed